lib/src/precompute_client.dart

```diff
@@ -0,0 +1,267 @@
+import 'dart:convert';
+
+import 'package:eppo/eppo_sdk.dart';
+import 'package:eppo/src/sdk_version.dart' as sdk;
+import 'package:logging/logging.dart';
+
+// Parameters for precomputed flags requests
+class SdkOptions {
+  /// SDK key for authentication
+  final String sdkKey;
+
+  /// Platform for the SDK
+  final sdk.SdkPlatform sdkPlatform;
```

Comment:
For now customers will provide the platform - when I move this to flutter I will introspect on the runtime and deduce this myself.